scripts/ts-utils/setup-enclave.ts

```diff
@@ -10,10 +10,11 @@ import colors from "colors";
 const transferAmount = "100000000000000";
 const enclaveAccount = process.argv[2];
 const mrenclave = process.argv[3];
-const accountPassword = process.argv[4];
+const block = process.argv[4];
+// const accountPassword = process.argv[4];
 
 //put account in private.json
-const private_account_pair = require("./private.json");
+// const private_account_pair = require("./private.json");
```

Comment:
Please remove commented lines if we don't need them :)

Reply:
Removed ✅